pootle/core/decorators.py

```diff
@@ -132,10 +132,8 @@ def set_resource(request, path_obj, dir_path, filename):
         resource_path = resource_path + filename
 
         try:
-            store = Store.objects.select_related(
-                'translation_project',
-                'parent',
-            ).get(pootle_path=pootle_path)
+            store = Store.objects.live() \
```

Comment:
Do you remove the `.select_related()` call because it's included in the manager by default?

If you check the docstring for the default manager you'll read *Mimics `select_related(depth=1)` behavior. Pending review.* So for now I'd leave `select_related()` here to avoid regressing whenever default managers are reviewed.